src/pack-install.js

```diff
@@ -34,7 +34,15 @@ const packInstall = function(pkgdir) {
         if (pack.status !== 0) {
           return Promise.reject(`npm pack failed:\n${pack.out}`);
         }
-        const tarball = getParsedJsonFromOutput(pack.stdout)[0].filename;
+
+        let tarball = getParsedJsonFromOutput(pack.stdout)[0].filename;
+
+        if (tarball.startsWith('@')) {
```

Comment:
nit: Should we leave a TODO saying we should remove this once we move all repos to Node 18?

Reply:
answered